scriptworker/worker.py

```diff
@@ -65,6 +66,9 @@
             except ScriptWorkerException as e:
                 status = worst_level(status, e.exit_code)
                 log.error("Hit ScriptWorkerException: {}".format(str(e)))
+            except (aiohttp.errors.DisconnectedError, aiohttp.errors.ClientError) as e:
+                status = worst_level(status, STATUSES['resource-unavailable'])
+                log.error("Hit aiohttp error: {}".format(str(e)))
```

Comment:
Nit: `str(e)` shouldn't be necessary as the conversion is already done by `format()`